ingestion/src/metadata/ingestion/source/presto.py

```diff
@@ -9,15 +9,69 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import re
 from urllib.parse import quote_plus
 
+from pyhive.sqlalchemy_presto import PrestoDialect
+from sqlalchemy import types, util
+from sqlalchemy.databases import mysql
+from sqlalchemy.engine import reflection
+
 from metadata.generated.schema.entity.services.databaseService import (
     DatabaseServiceType,
 )
 from metadata.ingestion.ometa.openmetadata_rest import MetadataServerConfig
 from metadata.ingestion.source.sql_source import SQLSource
 from metadata.ingestion.source.sql_source_common import SQLConnectionConfig
 
+_type_map = {
+    "boolean": types.Boolean,
+    "tinyint": mysql.MSTinyInteger,
```

Comment:
@ulixius9 why are we pointing this to mysql?

Reply:
sqlalchemy does not support tiny integer datatype, so the sqlalchemy_presto is internally referring to tiny integer datatype of mysql.